ServerlessLibraryAPI/wwwroot/items.json

```diff
@@ -536,7 +536,7 @@
     "language": "csharp",
     "type": "functionapp",
     "authortype": "Community",
-    "runtimeversion": "na"
+    "runtimeversion": "v2"
```

Comment:
as per the ARM deployment template, this should be v1

Reply:
Same thing here,s ays v2 : https://github.com/0GiS0/audio-azure-functions/blob/master/host.json